lib/screens/feedback_page.dart

```diff
@@ -0,0 +1,76 @@
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:flutter/material.dart';
+import 'package:innopolis_feedback/data.dart';
+import 'package:innopolis_feedback/shared/styles.dart';
+
+class FeedbackPage extends StatefulWidget {
+  final StudentFeedback f;
+
+  FeedbackPage(this.f);
+
+  @override
+  _FeedbackPageState createState() => _FeedbackPageState();
+}
+
+class _FeedbackPageState extends State<FeedbackPage> {
+  List<FeedbackComment> comments = [];
+  String uid, email;
+
+  updateComments(List<FeedbackComment> c) {
+    setState(() {
+      comments = c;
+    });
+  }
+
+  fetchComments() {
+    getComments(widget.f.feedbackId).listen((c) {
+      updateComments(c);
+    });
+  }
+
+  @override
+  void initState() {
+    fetchComments();
+    uid = FirebaseAuth.instance.currentUser.uid;
+    email = FirebaseAuth.instance.currentUser.email;
+    super.initState();
+  }
+
+
+  String prettyDate(DateTime date) =>
+      '${date.hour}:${date.minute} ${date.day}.${date.month}.${date.year}';
+
+  Widget commentToWidget(FeedbackComment c) => ListTile(
+        title: Text(c.email),
+        subtitle: Text(c.text),
+        trailing: Text(prettyDate(c.date)),
+      );
+
+  final commentHolder = TextEditingController();
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(title: Text(widget.f.email), centerTitle: true,),
+        body: Column(
+          children: [
+            Text(widget.f.message),
+            TextField(
```

Comment:
TextField does not expand if the comment is more than one line long.

Reply:
there are properties for this if I'm not mistaken

Reply:
Yes, check [here](https://stackoverflow.com/a/54348186/9405663).

Reply:
Helped, thanks